### PR TITLE
Additional work stock locations

### PR DESCRIPTION
**Pull request by bennsimon**
1. Work on Dynamic Tasking
2. Update Stock Endpoint
3. Update/Add String values for translation
4. Fix Service Point Pagination Bug
5. Update user location with actual geometry
6. Fix undo dialog message for product
7. Cancel Fix problem task if initial product task is undone

### CODE REVIEW

File: opensrp-eusm/src/main/java/org/smartregister/eusm/config/AppTaskingLibraryConfiguration.java [hunk 1]
```diff
@@ -95,9 +95,8 @@ public Float getLocationBuffer() {
     public void startImmediateSync() {
         LocationTaskServiceJob.scheduleJobImmediately(LocationTaskServiceJob.TAG);
         SyncServiceJob.scheduleJobImmediately(SyncServiceJob.TAG);
-        SyncStockTypeServiceJob.scheduleJobImmediately(SyncStockTypeServiceJob.TAG);
         ImageUploadServiceJob.scheduleJobImmediately(ImageUploadServiceJob.TAG);
-        SyncStockServiceJob.scheduleJobImmediately(SyncStockServiceJob.TAG);
+        //SyncStockServiceJob.scheduleJobImmediately(SyncStockServiceJob.TAG);
```
Comment (githengi) [inline opensrp-eusm/src/main/java/org/smartregister/eusm/config/AppTaskingLibraryConfiguration.java:99]: Why is this removed?

File: opensrp-eusm/src/main/java/org/smartregister/eusm/interactor/LoginInteractor.java [hunk 2]
```diff
@@ -27,9 +26,6 @@ protected void scheduleJobsPeriodically() {
         ImageUploadServiceJob.scheduleJob(ImageUploadServiceJob.TAG,
                 BuildConfig.PULL_UNIQUE_IDS_MINUTES, getFlexValue((int) BuildConfig.PULL_UNIQUE_IDS_MINUTES));
 
-        SyncStockServiceJob.scheduleJob(SyncStockServiceJob.TAG,
-                BuildConfig.PULL_UNIQUE_IDS_MINUTES, getFlexValue((int) BuildConfig.PULL_UNIQUE_IDS_MINUTES));
-
```
Comment (githengi) [inline opensrp-eusm/src/main/java/org/smartregister/eusm/interactor/LoginInteractor.java:32]: Why is this being removed?

File: opensrp-eusm/src/main/java/org/smartregister/eusm/config/AppTaskingLibraryConfiguration.java (hunk 1 above)
Comment (bennsimon) [reply opensrp-eusm/src/main/java/org/smartregister/eusm/config/AppTaskingLibraryConfiguration.java:99]: It should be called when the structures so as to populate the `locations` filter have been fetched so i moved the call to `AppLocationIntentService`

File: opensrp-eusm/src/main/java/org/smartregister/eusm/interactor/LoginInteractor.java (hunk 2 above)
Comment (bennsimon) [reply opensrp-eusm/src/main/java/org/smartregister/eusm/interactor/LoginInteractor.java:32]: Moved it to `AppLocationIntentService` so it will be called when the `LocationService` Job runs.
https://github.com/OpenSRP/opensrp-client-eusm/blob/additional-work-stock-locations/opensrp-eusm/src/main/java/org/smartregister/eusm/service/AppLocationTaskIntentService.java

**Review by githengi — APPROVED** [review]
